src/sentry/tasks/store.py

```diff
@@ -708,6 +708,9 @@ def _do_save_event(
             if cache_key:
                 with metrics.timer("tasks.store.do_save_event.delete_cache"):
                     processing.event_processing_store.delete_by_key(cache_key)
+        except Exception:
+            metrics.incr("events.save_event.exception", tags={"event_type": event_type})
+            raise
```

Comment:
Wouldn't a raised exception here be logged to sentry / sentry4sentry, giving you the error counts for free?

Reply:
So re-raising the exception would move the attribution for the exception? Right now this is just for counting exceptions that are thrown during `eventmanager.save()` pretty much. We wouldn't be handling any exceptions differently other than re-raising them from a different fire

Reply:
I think we specifically want this logged to datadog for use in their SLO feature.

Reply:
Ah I misread the comment. Yeah we want the datadog SLO stuff, not really about the metric itself. Thx!